packages/earth-admin/src/components/toggle/Toggle.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+import './styles.scss';
+
+interface IProps {
+  className?: string;
+  value?: boolean;
+  label?: string
+  onChange?: (e) => {};
+}
+
+export const Toggle = React.forwardRef((props: any, ref: any) => {
+  const {className, value, label, onChange, name, checked, ceva} = props;
```

Comment:
`name`, `checked` are not part of IProps and you should probably delete `ceva`

Also it seems like value, onChange, label and name should be required. If the code is not defensively checking for them it should be required. This surfaces errors during build time, not run time